cmd/agent/subcommands/run/command.go

```diff
@@ -640,7 +639,7 @@ func StopAgentWithDefaults(server dogstatsdServer.Component, agentAPI internalAP
 }
 
 // stopAgent Tears down the agent process
-func stopAgent(cliParams *cliParams, server dogstatsdServer.Component, agentAPI internalAPI.Component) {
+func stopAgent(cliParams *cliParams, _ dogstatsdServer.Component, agentAPI internalAPI.Component) {
```

Comment:
If the `dogstatsdServer.Component` is no longer used in this method, you can remove it from the list of arguments.